draft-packages/well/styles.scss

```diff
@@ -15,25 +15,37 @@
 
 .positive {
   background: $color-green-100;
-  border-color: var(#{$color-green-300-id}, $color-green-200);
+  border-color: var(#{$color-green-300-id}, $color-green-300);
```

Comment:
🤔 do you know why these have the verbose id thing and not just `border-color: $color-green-300`?

Reply:
Oh, looks like this was only in place so that it could be `green-200` if the CSS var didn't exist (to support Zen). Now that these are the same, this can be simplified.

Reply:
Ahh that's why. Will update now.